airflow/models/connection.py

```diff
@@ -35,6 +36,8 @@
 from airflow.utils.log.secrets_masker import mask_secret
 from airflow.utils.module_loading import import_string
 
+log = logging.getLogger()
```

Comment:
Use `getLogger(__file__)` to give the logger a name. Otherwise it returns the root logger, making the log messages very difficult to manage.